nodeup/pkg/model/packages.go

```diff
@@ -61,7 +61,6 @@ func (b *PackagesBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 		c.AddTask(&nodetasks.Package{Name: "ebtables"})
 		c.AddTask(&nodetasks.Package{Name: "ethtool"})
 		c.AddTask(&nodetasks.Package{Name: "iptables"})
-		c.AddTask(&nodetasks.Package{Name: "libcgroup"})
```

Comment:
Let's add a case for RHEL 9 and move the `libcgroup` to `default` case to keep behaviour for existing distros.

Reply:
I removed `switch`, there would be too much duplication (i.e. install libcgroup in `DistributionAmazonLinux2023:` in in `default:`)